llvm/lib/SYCLLowerIR/SYCLPropagateAspectsUsage.cpp

```diff
@@ -441,6 +446,94 @@ void createUsedAspectsMetadataForFunctions(
   }
 }
 
+void createConditionallyUsedAspectsMetadataForFunctions(
+    FunctionToConditionExpressionsAndConditionalAspectsTy
+        &FunctionToConditionExpressionsAndConditionalAspects,
```

Comment:
Those are pretty long words. I think that if we do `Function` -> `Func` and `Expression` -> `Expr` that would be slightly better but still readable. `Func` and `Expr` are pretty widespread in the codebase (especially in clang)